### DISCUSSION TITLE
grafana监控值不对

### DISCUSSION THREAD

**Discussion by tangpeibin123456** [apache/apisix#5398] [General]
1.用脚本 跑了100次 我的测试接口

![image](https://user-images.githubusercontent.com/13352114/139777911-9afc0c75-d4dc-44ed-8fa3-d6e8c605c4f5.png)

把100条日志 导入 表格  进行统计 可以得到平均值是0.008秒  也是8ms

![image](https://user-images.githubusercontent.com/13352114/139778476-35c59ccf-d3a5-48c3-945b-77cf427f9a21.png)
看监控 到了200ms   15 ms 为啥啊

<html xmlns:v="urn:schemas-microsoft-com:vml" xmlns:o="urn:schemas-microsoft-com:office:office" xmlns:x="urn:schemas-microsoft-com:office:excel" xmlns="http://www.w3.org/TR/REC-html40">
<head>

<meta name=Generator content="Microsoft Excel">

<style>

</style>
</head>
<body>

10.244.41.0 | - | - | [02/Nov/2021:02:32:59 | +0000] | api2.colorv.com | GET /v1/cloud/test HTTP/1.1 | 200 | 84 | 0.008 | - | curl/7.47.0 | 10.99.204.144:9000 | 200 | 0.004 | http://api2.colorv.com |   |  
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
10.244.9.0 | - | - | [02/Nov/2021:02:32:58 | +0000] | api2.colorv.com | GET /v1/cloud/test HTTP/1.1 | 200 | 84 | 0.008 | - | curl/7.47.0 | 10.99.204.144:9000 | 200 | 0.008 | http://api2.colorv.com |   |  
10.244.7.0 | - | - | [02/Nov/2021:02:32:58 | +0000] | api2.colorv.com | GET /v1/cloud/test HTTP/1.1 | 200 | 84 | 0.005 | - | curl/7.47.0 | 10.99.204.144:9000 | 200 | 0.004 | http://api2.colorv.com |   |  
10.244.5.0 | - | - | [02/Nov/2021:02:32:59 | +0000] | api2.colorv.com | GET /v1/cloud/test HTTP/1.1 | 200 | 84 | 0.007 | - | curl/7.47.0 | 10.99.204.144:9000 | 200 | 0.004 | http://api2.colorv.com |   |  
10.244.7.0 | - | - | [02/Nov/2021:02:33:00 | +0000] | api2.colorv.com | GET /v1/cloud/test HTTP/1.1 | 200 | 84 | 0.005 | - | curl/7.47.0 | 10.99.204.144:9000 | 200 | 0.012 | http://api2.colorv.com |   |  
10.244.46.0 | - | - | [02/Nov/2021:02:32:58 | +0000] | api2.colorv.com | GET /v1/cloud/test HTTP/1.1 | 200 | 84 | 0.008 | - | curl/7.47.0 | 10.99.204.144:9000 | 200 | 0.004 | http://api2.colorv.com |   |  
10.244.6.0 | - | - | [02/Nov/2021:02:32:59 | +0000] | api2.colorv.com | GET /v1/cloud/test HTTP/1.1 | 200 | 84 | 0.006 | - | curl/7.47.0 | 10.99.204.144:9000 | 200 | 0.008 | http://api2.colorv.com |   |  
10.244.3.0 | - | - | [02/Nov/2021:02:32:59 | +0000] | api2.colorv.com | GET /v1/cloud/test HTTP/1.1 | 200 | 84 | 0.007 | - | curl/7.47.0 | 10.99.204.144:9000 | 200 | 0.008 | http://api2.colorv.com |   |  
10.244.8.0 | - | - | [02/Nov/2021:02:33:00 | +0000] | api2.colorv.com | GET /v1/cloud/test HTTP/1.1 | 200 | 84 | 0.007 | - | curl/7.47.0 | 10.99.204.144:9000 | 200 | 0.008 | http://api2.colorv.com |   |  
10.244.3.0 | - | - | [02/Nov/2021:02:33:01 | +0000] | api2.colorv.com | GET /v1/cloud/test HTTP/1.1 | 200 | 84 | 0.005 | - | curl/7.47.0 | 10.99.204.144:9000 | 200 | 0.004 | http://api2.colorv.com |   |  
10.244.32.0 | - | - | [02/Nov/2021:02:33:00 | +0000] | api2.colorv.com | GET /v1/cloud/test HTTP/1.1 | 200 | 84 | 0.014 | - | curl/7.47.0 | 10.99.204.144:9000 | 200 | 0.004 | http://api2.colorv.com |   |  
10.244.45.0 | - | - | [02/Nov/2021:02:33:02 | +0000] | api2.colorv.com | GET /v1/cloud/test HTTP/1.1 | 200 | 84 | 0.007 | - | curl/7.47.0 | 10.99.204.144:9000 | 200 | 0.008 | http://api2.colorv.com |   |  
10.244.5.0 | - | - | [02/Nov/2021:02:33:01 | +0000] | api2.colorv.com | GET /v1/cloud/test HTTP/1.1 | 200 | 84 | 0.009 | - | curl/7.47.0 | 10.99.204.144:9000 | 200 | 0.008 | http://api2.colorv.com |   |  
10.244.26.0 | - | - | [02/Nov/2021:02:33:02 | +0000] | api2.colorv.com | GET /v1/cloud/test HTTP/1.1 | 200 | 84 | 0.006 | - | curl/7.47.0 | 10.99.204.144:9000 | 200 | 0.008 | http://api2.colorv.com |   |  
10.244.39.0 | - | - | [02/Nov/2021:02:33:01 | +0000] | api2.colorv.com | GET /v1/cloud/test HTTP/1.1 | 200 | 84 | 0.008 | - | curl/7.47.0 | 10.99.204.144:9000 | 200 | 0.008 | http://api2.colorv.com |   |  
10.244.15.0 | - | - | [02/Nov/2021:02:33:02 | +0000] | api2.colorv.com | GET /v1/cloud/test HTTP/1.1 | 200 | 84 | 0.007 | - | curl/7.47.0 | 10.99.204.144:9000 | 200 | 0.008 | http://api2.colorv.com |   |  
10.244.25.0 | - | - | [02/Nov/2021:02:33:04 | +0000] | api2.colorv.com | GET /v1/cloud/test HTTP/1.1 | 200 | 84 | 0.007 | - | curl/7.47.0 | 10.99.204.144:9000 | 200 | 0.008 | http://api2.colorv.com |   |  
10.244.46.0 | - | - | [02/Nov/2021:02:33:01 | +0000] | api2.colorv.com | GET /v1/cloud/test HTTP/1.1 | 200 | 84 | 0.007 | - | curl/7.47.0 | 10.99.204.144:9000 | 200 | 0.008 | http://api2.colorv.com |   |  
10.244.4.0 | - | - | [02/Nov/2021:02:33:04 | +0000] | api2.colorv.com | GET /v1/cloud/test HTTP/1.1 | 200 | 84 | 0.012 | - | curl/7.47.0 | 10.99.204.144:9000 | 200 | 0.012 | http://api2.colorv.com |   |  
10.244.11.0 | - | - | [02/Nov/2021:02:33:04 | +0000] | api2.colorv.com | GET /v1/cloud/test HTTP/1.1 | 200 | 84 | 0.007 | - | curl/7.47.0 | 10.99.204.144:9000 | 200 | 0.004 | http://api2.colorv.com |   |  
10.244.24.0 | - | - | [02/Nov/2021:02:33:02 | +0000] | api2.colorv.com | GET /v1/cloud/test HTTP/1.1 | 200 | 84 | 0.008 | - | curl/7.47.0 | 10.99.204.144:9000 | 200 | 0.004 | http://api2.colorv.com |   |  
10.244.17.0 | - | - | [02/Nov/2021:02:33:03 | +0000] | api2.colorv.com | GET /v1/cloud/test HTTP/1.1 | 200 | 84 | 0.005 | - | curl/7.47.0 | 10.99.204.144:9000 | 200 | 0.008 | http://api2.colorv.com |   |  
10.244.17.0 | - | - | [02/Nov/2021:02:33:03 | +0000] | api2.colorv.com | GET /v1/cloud/test HTTP/1.1 | 200 | 84 | 0.009 | - | curl/7.47.0 | 10.99.204.144:9000 | 200 | 0.012 | http://api2.colorv.com |   |  
10.244.42.0 | - | - | [02/Nov/2021:02:33:04 | +0000] | api2.colorv.com | GET /v1/cloud/test HTTP/1.1 | 200 | 84 | 0.008 | - | curl/7.47.0 | 10.99.204.144:9000 | 200 | 0.008 | http://api2.colorv.com |   |  
10.244.5.0 | - | - | [02/Nov/2021:02:33:03 | +0000] | api2.colorv.com | GET /v1/cloud/test HTTP/1.1 | 200 | 84 | 0.009 | - | curl/7.47.0 | 10.99.204.144:9000 | 200 | 0.008 | http://api2.colorv.com |   |  
10.244.37.0 | - | - | [02/Nov/2021:02:33:05 | +0000] | api2.colorv.com | GET /v1/cloud/test HTTP/1.1 | 200 | 84 | 0.007 | - | curl/7.47.0 | 10.99.204.144:9000 | 200 | 0.008 | http://api2.colorv.com |   |  
10.244.16.0 | - | - | [02/Nov/2021:02:33:05 | +0000] | api2.colorv.com | GET /v1/cloud/test HTTP/1.1 | 200 | 84 | 0.013 | - | curl/7.47.0 | 10.99.204.144:9000 | 200 | 0.012 | http://api2.colorv.com |   |  
10.244.38.0 | - | - | [02/Nov/2021:02:33:06 | +0000] | api2.colorv.com | GET /v1/cloud/test HTTP/1.1 | 200 | 84 | 0.032 | - | curl/7.47.0 | 10.99.204.144:9000 | 200 | 0.032 | http://api2.colorv.com |   |  
10.244.36.0 | - | - | [02/Nov/2021:02:33:06 | +0000] | api2.colorv.com | GET /v1/cloud/test HTTP/1.1 | 200 | 84 | 0.021 | - | curl/7.47.0 | 10.99.204.144:9000 | 200 | 0.02 | http://api2.colorv.com |   |  
10.244.33.0 | - | - | [02/Nov/2021:02:33:05 | +0000] | api2.colorv.com | GET /v1/cloud/test HTTP/1.1 | 200 | 84 | 0.004 | - | curl/7.47.0 | 10.99.204.144:9000 | 200 | 0.004 | http://api2.colorv.com |   |  
10.244.33.0 | - | - | [02/Nov/2021:02:33:05 | +0000] | api2.colorv.com | GET /v1/cloud/test HTTP/1.1 | 200 | 84 | 0.007 | - | curl/7.47.0 | 10.99.204.144:9000 | 200 | 0.008 | http://api2.colorv.com |   |  
10.244.46.0 | - | - | [02/Nov/2021:02:33:06 | +0000] | api2.colorv.com | GET /v1/cloud/test HTTP/1.1 | 200 | 84 | 0.007 | - | curl/7.47.0 | 10.99.204.144:9000 | 200 | 0.008 | http://api2.colorv.com |   |  
10.244.40.0 | - | - | [02/Nov/2021:02:33:07 | +0000] | api2.colorv.com | GET /v1/cloud/test HTTP/1.1 | 200 | 84 | 0.009 | - | curl/7.47.0 | 10.99.204.144:9000 | 200 | 0.008 | http://api2.colorv.com |   |  
10.244.24.0 | - | - | [02/Nov/2021:02:33:06 | +0000] | api2.colorv.com | GET /v1/cloud/test HTTP/1.1 | 200 | 84 | 0.008 | - | curl/7.47.0 | 10.99.204.144:9000 | 200 | 0.008 | http://api2.colorv.com |   |  
10.244.37.0 | - | - | [02/Nov/2021:02:33:07 | +0000] | api2.colorv.com | GET /v1/cloud/test HTTP/1.1 | 200 | 84 | 0.008 | - | curl/7.47.0 | 10.99.204.144:9000 | 200 | 0.008 | http://api2.colorv.com |   |  
10.244.26.0 | - | - | [02/Nov/2021:02:33:07 | +0000] | api2.colorv.com | GET /v1/cloud/test HTTP/1.1 | 200 | 84 | 0.007 | - | curl/7.47.0 | 10.99.204.144:9000 | 200 | 0.008 | http://api2.colorv.com |   |  
10.244.45.0 | - | - | [02/Nov/2021:02:33:07 | +0000] | api2.colorv.com | GET /v1/cloud/test HTTP/1.1 | 200 | 84 | 0.008 | - | curl/7.47.0 | 10.99.204.144:9000 | 200 | 0.008 | http://api2.colorv.com |   |  
10.244.32.0 | - | - | [02/Nov/2021:02:33:08 | +0000] | api2.colorv.com | GET /v1/cloud/test HTTP/1.1 | 200 | 84 | 0.008 | - | curl/7.47.0 | 10.99.204.144:9000 | 200 | 0.008 | http://api2.colorv.com |   |  
10.244.35.0 | - | - | [02/Nov/2021:02:33:09 | +0000] | api2.colorv.com | GET /v1/cloud/test HTTP/1.1 | 200 | 84 | 0.006 | - | curl/7.47.0 | 10.99.204.144:9000 | 200 | 0.004 | http://api2.colorv.com |   |  
10.244.46.0 | - | - | [02/Nov/2021:02:33:08 | +0000] | api2.colorv.com | GET /v1/cloud/test HTTP/1.1 | 200 | 84 | 0.015 | - | curl/7.47.0 | 10.99.204.144:9000 | 200 | 0.012 | http://api2.colorv.com |   |  
10.244.11.0 | - | - | [02/Nov/2021:02:33:08 | +0000] | api2.colorv.com | GET /v1/cloud/test HTTP/1.1 | 200 | 84 | 0.007 | - | curl/7.47.0 | 10.99.204.144:9000 | 200 | 0.008 | http://api2.colorv.com |   |  
10.244.46.0 | - | - | [02/Nov/2021:02:33:09 | +0000] | api2.colorv.com | GET /v1/cloud/test HTTP/1.1 | 200 | 84 | 0.009 | - | curl/7.47.0 | 10.99.204.144:9000 | 200 | 0.008 | http://api2.colorv.com |   |  
10.244.6.0 | - | - | [02/Nov/2021:02:33:10 | +0000] | api2.colorv.com | GET /v1/cloud/test HTTP/1.1 | 200 | 84 | 0.006 | - | curl/7.47.0 | 10.99.204.144:9000 | 200 | 0.004 | http://api2.colorv.com |   |  
10.244.36.0 | - | - | [02/Nov/2021:02:33:08 | +0000] | api2.colorv.com | GET /v1/cloud/test HTTP/1.1 | 200 | 84 | 0.006 | - | curl/7.47.0 | 10.99.204.144:9000 | 200 | 0.008 | http://api2.colorv.com |   |  
10.244.41.0 | - | - | [02/Nov/2021:02:33:09 | +0000] | api2.colorv.com | GET /v1/cloud/test HTTP/1.1 | 200 | 84 | 0.008 | - | curl/7.47.0 | 10.99.204.144:9000 | 200 | 0.008 | http://api2.colorv.com |   |  
10.244.35.0 | - | - | [02/Nov/2021:02:33:09 | +0000] | api2.colorv.com | GET /v1/cloud/test HTTP/1.1 | 200 | 84 | 0.005 | - | curl/7.47.0 | 10.99.204.144:9000 | 200 | 0.004 | http://api2.colorv.com |   |  
10.244.9.0 | - | - | [02/Nov/2021:02:33:10 | +0000] | api2.colorv.com | GET /v1/cloud/test HTTP/1.1 | 200 | 84 | 0.006 | - | curl/7.47.0 | 10.99.204.144:9000 | 200 | 0.004 | http://api2.colorv.com |   |  
10.244.7.0 | - | - | [02/Nov/2021:02:33:12 | +0000] | api2.colorv.com | GET /v1/cloud/test HTTP/1.1 | 200 | 84 | 0.007 | - | curl/7.47.0 | 10.99.204.144:9000 | 200 | 0.008 | http://api2.colorv.com |   |  
10.244.8.0 | - | - | [02/Nov/2021:02:33:11 | +0000] | api2.colorv.com | GET /v1/cloud/test HTTP/1.1 | 200 | 84 | 0.016 | - | curl/7.47.0 | 10.99.204.144:9000 | 200 | 0.016 | http://api2.colorv.com |   |  
10.244.37.0 | - | - | [02/Nov/2021:02:33:10 | +0000] | api2.colorv.com | GET /v1/cloud/test HTTP/1.1 | 200 | 84 | 0.006 | - | curl/7.47.0 | 10.99.204.144:9000 | 200 | 0.004 | http://api2.colorv.com |   |  

</body>

</html>

**Comment by tzssangglass** [comment]
the duration in grafana is around `10:44`, but in excel is around `02:33`,  does the times match between grafana and excel?

The 8 hour time zone difference between servers is understandable, the 8 hour and 10 minute difference is a little hard to understand.

And the indexs on grafana are P90, P95 and P99, they are not calculated as averages.

**Reply by tangpeibin123456** [reply]
时间是+8的  我的意思  就算取的P90, P95 and P99 差别太大了

**Reply by tzssangglass** [reply]
> 算取的P90, P95 and P99 差别太大了

I think you should first compare the raw data with grafana data at the same time

**Reply by juzhiyuan** [reply]
@tangpeibin123456 Hi, any update here? 😄
